testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -93,6 +93,16 @@ def test_dendrogram_produced(self):
         err_msg = "Dendrogram dictionary object was not produced"
         assert_(type(self.plot_data[1]) is dict, err_msg)
 
+    def test_dist_mat_to_vec_i_less_j(self):
+        """Test that distance is correct if i < j"""
+        err_msg = "dist_mat_to_vec function returning wrong values"
+        assert_(PSA.dist_mat_to_vec(5, 3, 4) is 9, err_msg)
```

Comment:
The `is` statement tests identity, not equality. In the present case, you should use the `==` comparison; even better in that specific context would be to use `assert_equal` rather than `assert_`. With `assert_equal` the traceback is more informative as it tells you what value you got and what value you expected.